src/sssom/util.py

```diff
@@ -194,6 +215,69 @@ def remove_mappings(self, msdf: "MappingSetDataFrame") -> None:
         self.clean_prefix_map()
 
 
+def _standardize_curie_or_iri(curie_or_iri: str, *, converter: Converter) -> str:
+    """Standardize a CURIE or IRI, returning the original if not possible.
+
+    :param curie_or_iri: Either a string representing a CURIE or an IRI
+    :returns:
+        - If the string represents an IRI, tries to standardize it. If not possible, returns the original value
+        - If the string represents a CURIE, tries to standardize it. If not possible, returns the original value
+        - Otherwise, return the original value
+    """
+    if is_iri(curie_or_iri):
+        return converter.standardize_uri(curie_or_iri) or curie_or_iri
+    if is_curie(curie_or_iri):
+        return converter.standardize_curie(curie_or_iri) or curie_or_iri
+    return curie_or_iri
+
+
+def _standardize_metadata(
+    converter: Converter, metadata: Dict[str, Any], *, raise_on_invalid: bool = False
+) -> None:
+    schema_object = _get_sssom_schema_object()
+    slots_dict = schema_object.dict["slots"]
```

Comment:
Not that it matters here for this PR, but "get_slot_list()" should probably be refactored (schema_object is not needed again)

Reply:
Yes, lots of stuff related to `_get_sssom_schema_object()` is begging for refactoring